### ISSUE TITLE
Page Publications

### ISSUE THREAD

**Issue by gplssm** [gplssm/egon-website#4]
- [ ] add page and 
- [ ] add exemplary data and include in page

**Comment by bmlancien** [comment]
Here are 2 different mockups for that page. I personally prefer the 1st one because each paper/talk has its own text and that would make for me more sense to describe each paper/talk than writing about the categories in general, like based on your mockup.

One other thing: I couldn't read the 4th link you wrote on your mockup (next to HTML). Maybe you can help me :)

![egon_publications_1](https://user-images.githubusercontent.com/33999424/81684739-84d77600-9457-11ea-9229-02dc12248f28.png)

![egon_publications_2](https://user-images.githubusercontent.com/33999424/81684758-89039380-9457-11ea-9265-4b00bd50f73c.png)

**Comment by gplssm** [comment]
I agree, the first option make more sense and is more inviting to the read to click one of the links. Vote: Option 1

The 4th link is called "bibtex". Me personally, I really like it when I can easily can access the paper metadata in bibtex format. For inspiration, look [here](https://gplssm.de/publications/). 

One minor design aspect: the icons seem to be quite small. Maybe it would an idea to increase their size and use them as anchors while they are overlapping the box' border. Just a quick idea. Don't take it too serious ;-)

Could you please implement your mockup. I'll take care about binding the content from a publications.yml file to it afterwards.

For the first month, I would show any publications nor talks, because we don't have any. As temporary solution I would simple write 1-2 sentences in normal text including a link to publications listed on the open_eGo page.
Later, when we have publications. I'll remove the text, fill the .yml file with content, and this should appear then...

**Comment by nesnoj** [comment]
Status: CSS not implemented yet

**Comment by nader-00** [comment]
Resolved via PR #105/#106
